packages/mobile/src/screens/product-info/components/price-block/lot-price-block.tsx

```diff
@@ -0,0 +1,80 @@
+import { ColorPalette, ProductDto } from '@vse-bude/shared';
+import React, { FC } from 'react';
+import { PlusSvg } from '~/assets/svg/plus';
+import { PrimaryButton, StarIcon, Text, View } from '~/components/components';
+import { useCustomTheme, useTranslation } from '~/hooks/hooks';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+const LotPriceBlock: FC<Pick<ProductDto, 'currentPrice' | 'minimalBid'>> = ({
+  currentPrice,
+  minimalBid,
+}) => {
+  const { colors } = useCustomTheme();
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <View
+        style={[
+          globalStyles.flexDirectionRow,
+          globalStyles.justifyContentCenter,
+          globalStyles.alignItemsCenter,
+          styles.currentBid,
+          { backgroundColor: colors.backgroundSecondary },
+        ]}
+      >
```

Comment:
You duplicating wrapper code. It might worth extracting it to a separate component

Reply:
fixed